.changeset/tasty-fishes-tease.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': major
```

Comment:
Since this doesn't introduce a breaking change, this should either be a `minor` or a `patch`.

Reply:
I thought net new atomics, components, were a major change and extensions of existing atomics components were a minor change?